### PR TITLE
update of readme file

### PR DESCRIPTION
**Pull request by felipanico**

### CODE REVIEW

**Comment by wesleyhf** [comment]
Duplicated of #16

File: README.md [hunk 1]
```diff
@@ -3,17 +3,41 @@
 [![Build Status](https://travis-ci.com/fireworkweb/fwd.svg?branch=php)](https://travis-ci.com/fireworkweb/fwd)
 [![codecov](https://codecov.io/gh/fireworkweb/fwd/branch/php/graph/badge.svg)](https://codecov.io/gh/fireworkweb/fwd)
 
+## Description
+
+fwd it's a command line docker interface to build web applications based in `.env` files
```
Comment (fabriciojs) [inline README.md]: we need to improve this

File: README.md [hunk 2]
```diff
@@ -3,17 +3,41 @@
 [![Build Status](https://travis-ci.com/fireworkweb/fwd.svg?branch=php)](https://travis-ci.com/fireworkweb/fwd)
 [![codecov](https://codecov.io/gh/fireworkweb/fwd/branch/php/graph/badge.svg)](https://codecov.io/gh/fireworkweb/fwd)
```
Comment (fabriciojs) [inline README.md]: let's add the FW logo image as @eduardosilvajobs did on his PR.

File: README.md [hunk 3]
```diff
@@ -3,17 +3,41 @@
...
 
+## Description
+
+fwd it's a command line docker interface to build web applications based in `.env` files
+
+## Pre Requisites
+
+Docker
```
Comment (fabriciojs) [inline README.md]: let's add more restrictive versions (i.e Docker Composer v1.24+)

File: README.md [hunk 4]
```diff
@@ -3,17 +3,41 @@
...
 curl -L https://github.com/fireworkweb/fwd/raw/php/builds/fwd -o /usr/local/bin/fwd
 chmod +x /usr/local/bin/fwd
 ```
+Inside of your project:
 
-## Usage
+Recommended configuration of your `.env` file:
+
+```bash
+DB_CONNECTION=mysql
```
Comment (fabriciojs) [inline README.md]: this is nice, and should prompt us as well to check the `.env` file upon `fwd install` to make sure these entries have the proper values (create a ticket for that or provide a PR please)

**Comment by fabriciojs** [comment]
This is too out dated, the latest 1.0-rc readme has been updated. Please refer any new improvements to README on 1.0-rc branch.

**Pull request closed without merging**